### PR TITLE
Live backed project reward states

### PR DESCRIPTION
**Pull request by eoji**
# What ❓
- Renamed `button_manage_pledge` to `button_pledge_manage` to match the naming convention of the other colors.
- For a live backed project:
  - Backed reward:
    - Your selected reward is focused when you open the rewards sheet.
    - The pledge button is blue and says `Manage your pledge`.
    - The checkmark is blue with a blue background.
  - Other available rewards:
    - The pledge button is green and says `Select this instead`.
- Added `RewardUtils.checkmarkBackgroundDrawable` that returns the drawable resource ID of the check background based on project status.
- Updated the output `backedRewardPosition` in `RewardFragmentViewModel` to also emit when the project is live.

# How to QA? 🤔
View a live project that you've backed (check your profile). (Note: The PR does not include any work in the `PledgeFragment`.)

# Story 📖
[Trello Story](https://trello.com/c/qupuDjcL/1413-reward-states-manage-your-pledge-backer-live-project)

# See 👀
device-2019-06-25-144802 2019-06-25 14_48_28

### CODE REVIEW

**Comment by eoji** [comment]
[Reward States -  Manage your pledge (backer, live project)](https://trello.com/c/qupuDjcL/1413-reward-states-manage-your-pledge-backer-live-project)

**Comment by Scollaco** [comment]
Hey @eoji , I started reviewing your PR and I noticed something:

- When I open the rewards sheet in order to manage my reward, the buttons of the other rewards say `Select this instead` regardless if it's available or not. I think it makes sense to only show this text for Rewards that are still available, showing `No longer available` otherwise.

File: app/src/main/java/com/kickstarter/libs/utils/RewardUtils.java [hunk 1]
```diff
@@ -23,6 +24,17 @@ private RewardUtils() {}
 
   public static final double MAX_REWARD_LIMIT = 2000000000;
 
+  /**
+   * Returns the drawable resource ID of the check background based on project status.
+   */
+  public static @DrawableRes int checkBackgroundDrawable(final @NonNull Project project) {
```
[the remark this answers is not in the document]
Comment (eoji) [reply app/src/main/java/com/kickstarter/libs/utils/RewardUtils.java]: Hmm, not sure I understand. This code depends on the `Reward` model specifically. `Checkout` is not a model, it's a feature.

File: app/src/main/java/com/kickstarter/viewmodels/HorizontalRewardViewHolderViewModel.kt [hunk 2]
```diff
@@ -157,13 +183,24 @@ interface HorizontalRewardViewHolderViewModel {
                     .filter { BackingUtils.isBacked(it.first, it.second) }
                     .compose(ignoreValues())
 
+            this.manageYourPledgeIsVisible = this.projectAndReward
+                    .filter { it.first.isLive && it.first.isBacking }
```
[the remark this answers is not in the document]
Comment (eoji) [reply app/src/main/java/com/kickstarter/viewmodels/HorizontalRewardViewHolderViewModel.kt]: this got deleted woohoo lol

**Pull request merged**